lib/ibm/cloud/sdk/iam.rb

```diff
@@ -1,13 +1,18 @@
+require_relative 'logging'
+
 module IBM
   module Cloud
     module SDK
       class IAM < BaseService
+        include Logging
+
         def endpoint
           "https://iam.cloud.ibm.com".freeze
         end
 
         def initialize(api_key)
           @api_key = api_key
+          RestClient.log = logger
```

Comment:
As-is the `apikey` provided in the header will be exposed in the logs.

e.g.:
```
[----] I, [2020-10-08T07:17:55.787736 #941468:9045c0]  INFO -- : RestClient.post "https://iam.cloud.ibm.com/identity/token", "grant_type=urn%3Aibm%3Aparams%3Aoauth%3Agrant-type%3Aapikey&apikey=12345", "Accept"=>"application/json", "Content-Length"=>"72", "Content-Type"=>"application/x-www-form-urlencoded", "User-Agent"=>"rest-client/2.1.0 (linux-gnu x86_64) ruby/2.5.8p224"
```

Reply:
PR to filter 'apikey' upstream:
https://github.com/ManageIQ/manageiq/pull/20675

Reply:
Merged